src/pages/HomePage/HomePage.jsx

```diff
@@ -1,9 +1,36 @@
-import "./Home.scss";
+import "./HomePage.scss";
+import { useEffect, useState } from 'react';
+import { getResponseContent } from '../../utils/openAIcall';
 
 const Home = () => {
+
+  const [aiResponseContent, setAiResponseContent] = useState("");
+
+  useEffect(() => {
+    const fetchResponse = async () => {
+
+      // awaits result of getResponseContent from utils, which makes the API call, before setting the state to that result
+      const responseContent = await getResponseContent([
```

Comment:
Try catch here.
I think a better variable name like `getOpenAIResponse` is more informative here.



Also, you should abstract out the messages to another folder/file. Probably something called "prompts.js". This HomePage.jsx is not especially concerned about the details of `getResponseContent`.

Reply:
We'll be sure to abstract messages away later when we put this call where it's going to go.